config/router-chatbot.js

```diff
@@ -132,12 +148,18 @@ router.post('/', (req, res) => {
       controller.debug(req, `sendMessage:${msg}`);
       controller.setCurrentCampaign(req.user, req.campaign_id);
 
-      mobilecommons.chatbot(req.body, mobilecommonsOip, msg);
+      let oip = process.env.MOBILECOMMONS_OIP_CAMPAIGNBOT;
+      if (controller.isCommand(req, 'member_support')) {
+        oip = process.env.MOBILECOMMONS_OIP_AGENTVIEW;
+      }
+
+      mobilecommons.chatbot(req.body, oip, msg);
 
       return res.send({ message: msg });
     })
     .catch(err => {
       controller.error(req, res, err);
+
```

Comment:
I was adding the extra line in attempt to make the `return` more readable (like above on L 158). Per http://eslint.org/docs/rules/newline-before-return -- there's no "hard and fast rule". I'm fine with removing, but maybe it could make sense for us to add one (a rule?)